### DISCUSSION TITLE
Two Miners on Two PCs with single wallet - low hash rate

### DISCUSSION THREAD

**Discussion by cdnbum88** [Lolliedieb/lolMiner-releases#589] [Q&A]
I see a few 2 gpu questions and this is similar but think different.
I have a 2 gpu mining rig with 2 AMD cards running ~84 MH/s. I have a second computer on same home network with a RTX2070 Super and it is running ~40 MH/s.
I am using ethermine.org for mining and I see my stats and they look to only be reflecting my mining rig and not my 3 gpu. It only shows one worker.
I see shares being found on both rigs, but just wondering if my second PC is really adding to my mining or just going thru the paces and not being accounted for. 
I would have anticipated to see ~120ish MH/s showing up in my stats or at least close to that and not numbers like my mining rig only.
Both are running on 1.28a. I did pick one east and one west configuration. Could that be an issue? Thinking not.

One wallet should be able to receive from 1 to many miners I would think??

Any suggestions would be greatly appreciated. I am still trying to learn and figure all this stuff out.

**Comment by strongwazz** [comment]
What do your two configs look like?

**Reply by cdnbum88** [reply]
Sorry, when you say configs, which ones are you referring too?

**Reply by strongwazz** [reply]
The bit whereever/however you call lolminer to start it, ie

```lolMiner --algo ETHASH --pool eth.2miners.com:2020 --user <address>``` etc

**Reply by cdnbum88** [reply]
I looked at my mining address and I was one letter off. Must not have cut and pasted properly.

Thanks for making me look back at that config as that was the culprit and now it is reflecting right.

Thanks for the assistance.

**Comment by strongwazz** [comment]
All good 👍
